nexus/types/src/external_api/params.rs

```diff
@@ -276,7 +276,7 @@ pub struct SiloCreate {
 pub struct UserCreate {
     /// username used to log in
     pub external_id: UserId,
-    /// password used to log in
+    /// describes how to set the user's login password
```

Comment:
```suggestion
    /// how to set the user's login password
```

maybe? in the spirit of
- less is more
- 80 columns in the CLI help menu